From 730cced85f7056086b9e2cd693b76ecafce3e97e Mon Sep 17 00:00:00 2001
From: Karthik Kambatla <kasha@cloudera.com>
Date: Fri, 10 May 2013 19:09:51 -0700
Subject: [PATCH 062/596] MR1: CLOUDERA-BUILD. CDH-10752. Remove aspectj
 dependency in MR1 (cherry picked from commit
 edecc71e3d1edf9380f949647c35f3cf8436bd16)

(cherry picked from commit 1097cb26751813b1debdaf6f27a1d7b3c7d4d6b3)
(cherry picked from commit 9b85b3e579cdad75805015f377133abc70190c72)
(cherry picked from commit 9041171032ea1d5f06cf39d9736dcff53bee0cf5)
---
 hadoop-mapreduce1-project/ivy.xml |    5 -----
 1 file changed, 5 deletions(-)

diff --git a/hadoop-mapreduce1-project/ivy.xml b/hadoop-mapreduce1-project/ivy.xml
index 46276d9..425def6 100644
--- a/hadoop-mapreduce1-project/ivy.xml
+++ b/hadoop-mapreduce1-project/ivy.xml
@@ -121,11 +121,6 @@
    <dependency org="org.vafer" name="jdeb" rev="${jdeb.version}" conf="package->master"/>
    <dependency org="org.mortbay.jetty" name="jetty-servlet-tester" rev="${jetty.version}"
                conf="test->default"/>
-   <!-- dependency addition for the fault injection -->
-   <dependency org="org.aspectj" name="aspectjrt" rev="${aspectj.version}"
-               conf="compile->default"/>
-   <dependency org="org.aspectj" name="aspectjtools" rev="${aspectj.version}"
-               conf="compile->default"/>
    <dependency org="org.apache.zookeeper" name="zookeeper" rev="${zookeeper.version}"
                conf="compile->default"/>
    <dependency org="org.apache.zookeeper" name="zookeeper" rev="${zookeeper.version}"
-- 
1.7.9.5

